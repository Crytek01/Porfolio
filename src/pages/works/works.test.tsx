import { shallowRender } from '@common/utils/test/shallowRender';
import { describe, expect, it } from 'vitest';

import WorksPage from './works.page';

describe('Works page component Test', () => {
  it('renders correctly', () => {
    const renderer = shallowRender(<WorksPage />);

    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
