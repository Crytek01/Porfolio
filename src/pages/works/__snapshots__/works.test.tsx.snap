// Vitest Snapshot v1

exports[`Works page component Test > renders correctly 1`] = `
<ForwardRef(MotionComponent)
  animate="animate"
  exit="exit"
  initial="initial"
  transition={
    {
      "type": "just",
    }
  }
  variants={
    {
      "animate": {
        "opacity": 1,
        "y": 0,
      },
      "exit": {
        "opacity": 0,
        "y": 100,
      },
      "initial": {
        "opacity": 0,
        "y": 100,
      },
    }
  }
>
  <WorksPage />
</ForwardRef(MotionComponent)>
`;
